code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1233,34 +1232,6 @@
 				I = new organ(H) //Create the organ inside the player.
 				I.insert(H)
 
-/mob/living/carbon/human/revive()
-
-	if(species && !(species.flags & NO_BLOOD))

```

Comment:
Uh, shouldn't have gotten rid of all of `human/revive`. Sorry if I was unclear; it was only this that needed to be removed:

```
if(species && !(species.flags & NO_BLOOD))
	restore_blood()
```